arkane/ess/molpro.py

```diff
@@ -56,8 +56,26 @@ class MolproLog(ESSAdapter):
     MolproLog is an adapter for the abstract class ESSAdapter.
     """
 
-    def __init__(self, path):
+    def __init__(self, path, check_for_errors=True):
         self.path = path
+        if check_for_errors:
+            self.check_for_errors()
+
+    def check_for_errors(self):
+        """
+        Checks for common errors in a Molpro log file.
+        If any are found, this method will raise an error and crash.
+        """
+        with open(os.path.join(self.path), 'r') as f:
+            lines = f.readlines()
+            error = None
+            for line in reversed(lines):
+                # check for common error messages
+                if 'No convergence' in line:
```

Comment:
I am not familiar with `Molpro` better to have someone else. I found a few more cases in ARC, maybe we can add some of them as well?

Reply:
Good point. I'll add more examples from ARC's `trsh.py`